java/flight/flight-core/src/main/java/org/apache/arrow/flight/SetSessionOptionsResult.java

```diff
@@ -45,12 +45,14 @@ static ErrorValue fromProtocol(Flight.SetSessionOptionsResult.ErrorValue s) {
       return values()[s.getNumber()];
     }
 
+    @SuppressWarnings("EnumOrdinal")
     Flight.SetSessionOptionsResult.ErrorValue toProtocol() {
       return Flight.SetSessionOptionsResult.ErrorValue.values()[ordinal()];
     }
   }
 
   /** Per-option extensible error response container. */
+  @SuppressWarnings("JavaLangClash")
```

Comment:
What was the problem here?

Reply:
It's clashing with `java.lang.Error` https://docs.oracle.com/javase/8/docs/api/java/lang/Error.html

Reply:
+1 for renaming it, if it's not an external API change in some way

Reply:
This is public, it will be a breaking change.

Reply:
Right, the issue is as @danepitkin mentioned. Shall we leave the warning suppressed?

Reply:
Yes, let's leave the suppression in place.